OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/Analytics.java

```diff
@@ -622,6 +622,14 @@ void trackOpenInBrowserBannerEvent(@NonNull String eventName, @NonNull String bi
      */
     void trackVideoDownloadQualityChanged(@NonNull VideoQuality selectedVideoQuality, @NonNull VideoQuality oldVideoQuality);
 
+    /**
+     * Track event triggered on
+     *
+     * @param eventName Name of the event
+     * @param biValue   BiValue of the event
+     */
+    void trackAccountRegistrationEvent(@NonNull String eventName, @NonNull String biValue);
```

Comment:
As we are passing only two values 1- eventName 2-BiValue, we can make a generic function for this as previously there are methods available in which we are passing the same parameters.
i.e, `trackProfileScreenEvent` or `trackVideoDownloadQualityClicked`
Update its implementation accordingly

Reply:
I have refactored the code accordingly.